src/antsibull_changelog/cli.py

```diff
@@ -239,6 +239,11 @@ def create_argparser(program_name: str) -> argparse.ArgumentParser:
     release_parser.set_defaults(func=command_release)
     release_parser.add_argument("--version", help="override release version")
     release_parser.add_argument("--codename", help="override/set release codename")
+    release_parser.add_argument(
+        "--update-galaxy-file",
+        action="store_true",
+        help="override/set version in galaxy.yml",
```

Comment:
```suggestion
        help="re-write galaxy.yml with the supplied version. Note that this will remove comments and formatting",
```
(This likely needs a line-break somewhere.)